packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -238,6 +259,43 @@ export const getOptionPerUnit = (unit: string, show_tick_from_5: boolean): { val
         }));
     };
 
+    const { intraday, tick, daily } = duration_min_max;
+
+    const unitConfig: Record<
+        string,
+        { start: number; end: number; label: React.ReactNode } | (() => { value: number; label: React.ReactNode }[][])
+    > = {
+        m: {
+            start: Math.max(1, intraday?.min / 60),
+            end: Math.min(59, intraday?.max / 60),
+            label: <Localize i18n_default_text='min' />,
+        },
+        s: {
+            start: Math.max(15, intraday?.min),
```

Comment:
These numbers are from prod?

Reply:
Yeah its from production , so here what I am doing is , i am checking from production value `intraday`  . Intraday include min and max for seconds , minutes and hours (in seconds). 
we have upper limit for each seconds , minutes and hour . for eg seconds can't be greater then 59 . I am checking if the value from prod is lesser then to displat that value and if its more then just stick to the limit of that particular duration 

minutes -> 59
second-> 59
hours -> 24 hours (that is managed via minutes .)

same is for days and ticks